cms/djangoapps/contentstore/management/commands/tests/test_force_publish.py

```diff
@@ -0,0 +1,108 @@
+"""
+Tests for the force_publish management command
+"""
+import mock
+from django.core.management.base import CommandError
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+from contentstore.management.commands.force_publish import Command
+from contentstore.management.commands.utils import get_course_versions
+
+
+class TestForcePublish(SharedModuleStoreTestCase):
+    """
+    Tests for the force_publish management command
+    """
+    def setUp(self):
+        super(TestForcePublish, self).setUp()
+        self.course = CourseFactory.create(default_store=ModuleStoreEnum.Type.split)
```

Comment:
I think you'd want to create the course in a `setUpClass(cls)` classmethod, not in this `setUp` method. See http://swampcastle.org/2015/07/28/writing-faster-modulestore-tests.html